uniflow/op/model/lm/model.py

```diff
@@ -58,10 +59,7 @@ def _deserialize(self, data: List[str]) -> List[Dict[str, Any]]:
         Returns:
             List[Dict[str, Any]]: Deserialized data.
         """
-        return {
-            RESPONSE: data,
-            ERROR: "No errors.",
-        }
+        return [Context(context=data)]
```

Comment:
qq: why are you using a list here instead of directly returning Context(context=data)?

Reply:
I did this because LmModel._serialize takes in List[Context]